app/models/user.rb

```diff
@@ -24,6 +25,10 @@ def new_token
     end
   end
 
+  def feed
+   Micropost.where "user_id = ?", id
```

Comment:
nên viết scope tìm kiếm ở model micropost rồi gọi từ class Micropost và truyền đối số user_id vào để dùng nha em. Tha cho method feed trong model user như hiện tại
Trong model micropost
```suggestion
   scope :by_author, ->(user_id){where user_id: user_id}
```